Neos.ContentRepository/Classes/Security/Authorization/Privilege/Node/CreateNodePrivilegeContext.php

```diff
@@ -19,17 +20,37 @@
 class CreateNodePrivilegeContext extends NodePrivilegeContext
 {
     /**
-     * @var string
+     * @var NodeTypeManager
+     *
+     * @Flow\Inject
```

Comment:
&lt;nitpick&gt;According to our [CGL](https://flowframework.readthedocs.io/en/stable/TheDefinitiveGuide/PartV/CodingGuideLines/PHP.html#overview-of-documentation-annotations) the inject annotation should be above the `@var`:
```
/**
 * @Flow\Inject
 * @var NodeTypeManager
 */
```
&lt;/nitpick&gt;

Reply:
Oh, I never read [that part](https://flowframework.readthedocs.io/en/stable/TheDefinitiveGuide/PartV/CodingGuideLines/PHP.html#overview-of-documentation-annotations) with the order of the annotations in mind. Perhaps I didn't read id hard enough 📖😄.

It isn't explicitly stated in the CGL, but I'm used to separating most annotation types (especially `@var`, `@return`, `@throws`) with an empty line and most of the code I see in this project all annotations seem to be crammed together 🤷‍♂️.